ts/src/pro/binance.ts

```diff
@@ -169,7 +169,7 @@ export default class binance extends binanceRest {
         return newValue;
     }
 
-    stream (type: Str, subscriptionHash: Str, numSubscriptions = 1) {
+    streamId (type: Str, subscriptionHash: Str, numSubscriptions = 1) {
         const streamBySubscriptionsHash = this.safeDict (this.options, 'streamBySubscriptionsHash', this.createSafeDictionary ());
```

Comment:
@pcriadoperez maybe we can add inside `has` `subscribeOrderBook/subscribeTrades` etc so users can easily check if the exchange in question support is

Reply:
I thought about it, but as it's only adding streamProduce on any new handlers I think we should say that all watchFunctions support a their subscribe counterpart